Native SimNet.md

```diff
@@ -0,0 +1,227 @@
+We recommend to use the [xud-docker](User%20Guide.md) setup to trade on the `xud-simnet`; it does not require installation of `xud` or any dependencies. For a native installation, installing various dependencies of underlying clients, follow the guide below.
+
+## Goal
+
+This guide describes how to setup `xud` and connect to and trade on the XUD Simulation Network (`xud-simnet`). It should take about 15 minutes to complete all steps.
+
+`xud` is in alpha stage and the `xud-simnet` aims to give interested individuals an early 'look & feel' of how things will work lateron with a convenient, mostly automated setup. The `xud-simnet` uses magic simnet coins at all times.
+
+Once the setup of `xud-simnet` is completed, you will be able to query pending orders, place orders, and experiment with `xud`'s rich set of commands (`xucli --help`). When a match is found in the network, your orders will automatically be executed & settled instantly via an atomic swap. `xud-simnet` currently supports atomic swaps between BTC, LTC & ERC20 tokens. This guide will show how to install dependencies like `go` and how to use the `xud-simnet` to automatically setup bitcoin, litecoin & ethereum daemons, lightning daemons (LND) for bitcoin (BTC) and litecoin (LTC), raiden for ERC20 tokens, `xud` and finally connect to `xud-simnet`. This guide is written for Linux, macOS and Windows' [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-install) and suitable for everyone that knows how to use a command line.
+
+## Describing the Setup
+
+`xud-simnet` spins up several components on your machine which together provide support for exchanging order information and executing atomic swaps between BTC and LTC on the lightning network. It will setup the following components:
+
+- `neutrino` - light node, connecting to our remote btcd instance (which is mining the bitcoin simnet chain)
+- `ltcd` - local full node, connected to the litecoin simnet chain
+- `geth` - connecting to our remote geth instance (which is mining the ethereum simnet chain)
+- `lndbtc` - payment channel implementation for the bitcoin network
+- `lndltc` - payment channel implementation for the litecoin network
+- `raiden` - payment channel implementation for the ethereum network
+- `xud` - Exchange Union's decentralized exchange layer, managing orders and clients above
+
+## Installation 
+
+### Prerequisites
+
+Make sure you have the below programs installed:
+- [git](https://gist.github.com/derhuerst/1b15ff4652a867391f03#file-linux-md)
+- [make](https://www.cyberciti.biz/faq/debian-linux-install-gnu-gcc-compiler/)
```

Comment:
`sudo apt install make` will install `make` without all of the other stuff that is included in `build-essentials`.

Reply:
Adjusted.